Source/DafnyLanguageServer/Workspace/TextDocumentLoader.cs

```diff
@@ -338,14 +337,14 @@ bool loadCanceled
     protected virtual VerificationProgressReporter CreateVerificationProgressReporter(DafnyDocument document) {
       return new VerificationProgressReporter(
         loggerFactory.CreateLogger<VerificationProgressReporter>(),
-        document, notificationPublisher, diagnosticPublisher);
+        document, notificationPublisher, NotificationPublisher);
```

Comment:
Is there a way not to have two variables with the same name but different capitalization? This sounds like there is room for improvement.